### PR TITLE
Proposed fix for issue 10821

### PR DESCRIPTION
**Pull request by FalkoBabbage**
### Link to issue number:
10821
### Summary of the issue:
 list items are double speaking when UIA is enabled for office applications (https://github.com/nvaccess/nvda/issues/10821). I used the scripts already implemented in NVDAObjects/IAccessible/winword.py to skip the bullet points of a list. Only adjustment is the use of UIA constant TextUnit_Paragraph.
### Description of user facing changes
control+upArrow and control+downArrow works as expected when in lists.
### Description of development approach
script_nextParagraph and script_previousParagraph are added to NVDAObjects/UIA/wordDocument.py like is done in NVDAObjects/IAccessible/winword.py.
### Testing strategy:
tested on Word 365 (2208 Build 16.0.15601.20148) 64 bits  with a bullet list and some paragraphs.
### Change log entries:
New features
Changes
Bug fixes
For Developers

### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [ ] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [ ] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] Security precautions taken.

### CODE REVIEW

File: source/NVDAObjects/UIA/wordDocument.py [hunk 1]
```diff
@@ -102,7 +102,8 @@ def getCommentInfoFromPosition(position):
 		typeID = UIAElement.GetCurrentPropertyValue(UIAHandler.UIA_AnnotationAnnotationTypeIdPropertyId)
 		# Use Annotation Type Comment if available
 		if typeID == UIAHandler.AnnotationType_Comment:
-			comment = UIAElement.GetCurrentPropertyValue(UIAHandler.UIA_NamePropertyId)
+			log.info("getting comment with name?")
```
Comment (leonardder) [inline source/NVDAObjects/UIA/wordDocument.py:105]: Could you please remove all unrelated changes?

**Comment by cary-rowen** [comment]
@FalkoBabbage 
Hi, any progress on this? Can the suggested changes be applied?

**Comment by seanbudd** [comment]
Closing this as abandoned
